.github/workflows/ci.yml

```diff
@@ -131,10 +131,19 @@ jobs:
         uses: "networktocode/gh-action-setup-poetry-environment@v6"
         with:
           poetry-version: "2.1.3"
+          python-version: "${{ matrix.python-version }}"
```

Comment:
Was this causing problems? I think we omitted this because this job runs all of the tests in docker so we weren't concerned with the local python version.